src/main/java/com/datapipe/jenkins/vault/credentials/AbstractVaultTokenCredentialWithExpiration.java

```diff
@@ -44,13 +44,14 @@ public void setUsePolicies(Boolean usePolicies) {
         this.usePolicies = usePolicies;
     }
 
-    private transient Map<String, Calendar> tokenExpiry;
+    // Renamed to prevent XStream from attempting to deserialize old instances of this class which had the type `Calendar` prior to https://github.com/jenkinsci/hashicorp-vault-plugin/pull/223.
+    private transient Map<String, Calendar> tokenExpiry2;
```

Comment:
Can we name the field tokenExpiryCache instead? I worry the 2 will cause confusion and someone looking for the 1 version of the field

Reply:
Renamed in [a309a65](https://github.com/jenkinsci/hashicorp-vault-plugin/pull/336/commits/a309a651c51ebad9fe2998be68212d1180ee6f56).

Reply:
Great, thanks!